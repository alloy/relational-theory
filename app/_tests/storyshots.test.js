"use strict";
var storyshots_1 = require("storyshots");
storyshots_1["default"]({
    storyRegex: /^((?!(r|R)elay).)*$/
});
