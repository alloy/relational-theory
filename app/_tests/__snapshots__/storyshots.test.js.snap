exports[`Storyshots Artist Header For Stubbed Data 1`] = `
<div>
  <h1>
    Another Example
  </h1>
  <div>
    <i>
      OK, b. 1999
    </i>
  </div>
  <i>
    12 Followers
  </i>
</div>
`;
